cmd/plural/template.go

```diff
@@ -30,35 +33,67 @@ func testTemplate(c *cli.Context) error {
 			continue
 		}
 
-		ctx := installation.Context
-		tmpl, err := template.MakeTemplate(string(testTemplate))
+		var errList []error
+		vals := genDefaultValues(conf, installation)
+
+		output, err := luaTmpValues(string(testTemplate), vals)
```

Comment:
Perhaps we should have a flag that tells what template type it is?

Reply:
brute forcing isn't ideal and we definitely will know server-side

Reply:
I will add flag for the type

Reply:
done